doc/man/8/ceph-bluestore-tool.rst

```diff
@@ -0,0 +1,123 @@
+:orphan:
+
+======================================================
+ ceph-bluestore-tool -- bluestore administrative tool
+======================================================
+
+.. program:: ceph-bluestore-tool
+
+Synopsis
+========
+
+| **ceph-bluestore-tool** *command*
+  [ --dev *device* ... ]
+  [ --path *osd path* ]
+  [ --out-dir *dir* ]
+  [ --log-file | -l *filename* ]
+  [ --deep ]
+| **ceph-bluestore-tool** fsck|repair --path *osd path* [ --deep ]
+| **ceph-bluestore-tool** show-label --dev *device* ...
```

Comment:
show-label also accepts --path arguments, for example: 
./bin/ceph-bluestore-tool --command show-label --path  ./dev/osd0/

Reply:
good call, we can fix that on master though and backport on top of this